drivers/display/uc81xx.c

```diff
@@ -654,6 +631,122 @@ static int uc81xx_init(const struct device *dev)
 	return uc81xx_controller_init(dev);
 }
 
+#if DT_HAS_COMPAT_STATUS_OKAY(ultrachip_uc8175)
+static int uc8175_set_cdi(const struct device *dev, bool border)
+{
+	const struct uc81xx_config *config = dev->config;
+	const struct uc81xx_data *data = dev->data;
+	const struct uc81xx_profile *p = config->profiles[data->profile];
+	uint8_t cdi = UC8175_CDI_VBD1 | UC8175_CDI_DDX0 | (p ? (p->cdi & UC8175_CDI_CDI_MASK) : 0);
+
+	if (!p || !p->override_cdi) {
+		return 0;
+	}
+
+	if (!border) {
+		/* Floating border */
+		cdi |= UC8175_CDI_VBD1 | UC8175_CDI_VBD0;
+	}
+
+	LOG_DBG("CDI: %#hhx", cdi);
+	return uc81xx_write_cmd_uint8(dev, UC81XX_CMD_CDI, cdi);
+}
```

Comment:
If the differences are too great and uc8179_set_cdi() cannot be used, it should at least be implemented in the same way as uc8179_set_cdi().

Reply:
yep, I just check against the datasheet and it's completely fine to use uc8176_set_cdi for uc8175, the extra bit in the CDI mask will be silently ignored by the chip, all good.